src/util/common.ts

```
@@ -89,7 +89,9 @@ export function sumSingleBalance(
     else
       throw new Error('Invalid balance value:' + balance + ' ' + token)
   }
-
+  if (!token || token.trim() === '') {
```

I dont think that is what is happening here. also, we should throw an error here? Cant think of a scenario without token & chain info